ui_framework/components/tabs/tabs.test.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { render, shallow } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+import sinon from 'sinon';
+
+import {
+  KuiTabs,
+} from './tabs';
+
+import {
+  KuiTab,
+} from './tab';
+
+describe('KuiTabs', () => {
+  const tabs = [
+    'Cobalt',
+    'Dextrose',
+    'Helium-3',
+    'Monosodium Glutamate',
+  ];
+
+  describe('throws an error', () => {
```

Comment:
@weltenwort We have similar error-related tests in https://github.com/elastic/kibana/blob/master/ui_framework/components/accessibility/keyboard_accessible.test.js#L16, where we could also implement your suggestion.

Before we move forward with your suggestion, I think we should define best practices for defining propType functions and associating them with components. Should we export them from their own module? Should they live within the same directory as the component which uses them? Should we group them together in a `prop_types` directory?

For now, I think we should keep these tests as they are, for the sake of simplicity and consistency, and then refactor all of the propType functions once we've defined best practices. What do you think?